1.13.2: missig pkgconfig dependency

Just found that webkitgtk failed on missing xkbcommon. header file.
```console
In file included from /usr/include/wpe-1.0/wpe/wpe.h:38,
                 from /usr/include/wpe-1.0/wpe/wpe-egl.h:37,
                 from /home/tkloczko/rpmbuild/BUILD/webkitgtk-2.36.3/Source/WebCore/platform/graphics/egl/GLContextEGLLibWPE.cpp:39:
/usr/include/wpe-1.0/wpe/input-xkb.h:48:10: fatal error: xkbcommon/xkbcommon.h: No such file or directory
   48 | #include <xkbcommon/xkbcommon.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
And indeed wpe-1.pc file has no `Requires: xkbcommon` line.
```console
[tkloczko@devel-g2v SPECS]$ rpm -ql libwpe-devel | grep include.*.h$ | xargs grep -h \#include|sort | uniq
#include "export.h"
#include "gamepad.h"
#include "input.h"
#include "input-xkb.h"
#include "keysyms.h"
#include "libwpe-version.h"
#include "loader.h"
#include "pasteboard.h"
#include "renderer-backend-egl.h"
#include "renderer-host.h"
#include <stdbool.h>
#include <stdint.h>
#include "version-deprecated.h"
#include "version.h"
#include "view-backend.h"
#include <wpe/export.h>
#include "wpe.h"
#include <xkbcommon/xkbcommon-compose.h>
#include <xkbcommon/xkbcommon.h>
```
As wpe iuses xkbcommon headers it should depend on in pkgconfig description

> This issue is not about --cflags output.

The issue is about resolving:
```
/usr/include/wpe-1.0/wpe/input-xkb.h:48:10: fatal error: xkbcommon/xkbcommon.h: No such file or directory
   48 | #include <xkbcommon/xkbcommon.h>
```

Which means the issue is about making sure that `pkg-config --cflags wpe-1.0` produces the correct flags to make header dependencies on `xkbcommon/xkbcommon.h` resolve. i.e. this is very much about --cflags output.

--- a/include/wpe/input-xkb.h
+++ b/include/wpe/input-xkb.h
@@ -45,12 +45,16 @@
 
 #include <stdbool.h>
 #include <stdint.h>
-#include <xkbcommon/xkbcommon.h>
-#include <xkbcommon/xkbcommon-compose.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+struct xkb_compose_state;
+struct xkb_compose_table;
+struct xkb_context;
+struct xkb_keymap;
+struct xkb_state;
 
 struct wpe_input_xkb_context;
 

--- a/wpe.pc.in
+++ b/wpe.pc.in
@@ -6,6 +6,6 @@ libdir=${exec_prefix}/lib
 Name: wpe-@WPE_API_VERSION@
 Description: The wpe library
 Version: @PROJECT_VERSION@
-Requires: @WPE_PC_REQUIRES@
+Requires.private: @WPE_PC_REQUIRES@
 Cflags: -I${includedir}/wpe-@WPE_API_VERSION@ @WPE_PC_CFLAGS@
 Libs: -L${libdir} -lwpe-@WPE_API_VERSION@

